packages/sync-server/src/app-gocardless/banks/tests/spk_marburg_biedenkopf_heladef1mar.spec.js

```diff
@@ -196,14 +196,14 @@ describe('SpkMarburgBiedenkopfHeladef1mar', () => {
         SpkMarburgBiedenkopfHeladef1mar.normalizeTransaction(tx, true),
       );
       const originalOrder = Array.from(normalizeTransactions);
+      console.log(originalOrder);
       const swap = (a, b) => {
         const swap = normalizeTransactions[a];
         normalizeTransactions[a] = normalizeTransactions[b];
         normalizeTransactions[b] = swap;
       };
-      swap(1, 4);
-      swap(3, 6);
-      swap(0, 7);
```

Comment:
Not sure how this test was passing before, as all these indices are out of range. Changed them to in-range